node.js

```diff
@@ -71,129 +92,171 @@ function isInModuloRange(input_value, lower_bound, include_lower, upper_bound, i
             // start < end
             return (input_value > lower_bound && input_value < upper_bound);
         }
-    }    
-}  
+    }
+}
 
 function summary(_, callback) {
-  console.log("vvvvv     vvvvv     Summary     vvvvv     vvvvv");
-  console.log("FingerTable: \n", FingerTable);
-  console.log("Predecessor: ", predecessor);
-  console.log("^^^^^     ^^^^^     End Summary     ^^^^^     ^^^^^")
-  callback(null, _self);
+    console.log("vvvvv     vvvvv     Summary     vvvvv     vvvvv");
+    console.log("FingerTable: \n", FingerTable);
+    console.log("Predecessor: ", predecessor);
+    console.log("^^^^^     ^^^^^     End Summary     ^^^^^     ^^^^^")
+    callback(null, _self);
 }
 
 function fetch({ request: { id } }, callback) {
-  console.log(`Requested User ${id}`);
-  if (!users[id]) {
-    callback({ code: 5 }, null); // NOT_FOUND error
-  } else {
-    callback(null, users[id]);
-  }
+    console.log(`Requested User ${id}`);
+    if (!users[id]) {
+        callback({ code: 5 }, null); // NOT_FOUND error
+    } else {
+        callback(null, users[id]);
+    }
 }
 
 function insert({ request: user }, callback) {
-  if (users[user.id]) {
-    const message = `Err: ${user.id} already exits`;
-    console.log(message);
-    callback({ code: 6, message }, null); // ALREADY_EXISTS error
-  } else {
-    users[user.id] = user;
-    const message = `Inserted User ${user.id}:`;
-    console.log(message);
-    callback({ status: 0, message }, null);
-  }
+    if (users[user.id]) {
+        const message = `Err: ${user.id} already exits`;
+        console.log(message);
+        callback({ code: 6, message }, null); // ALREADY_EXISTS error
+    } else {
+        users[user.id] = user;
+        const message = `Inserted User ${user.id}:`;
+        console.log(message);
+        callback({ status: 0, message }, null);
+    }
 }
 
-/* added 20191102 */
+/**
+ * Directly implement the pseudocode's find_successor() method.
+ * 
+ * However, it is able to discern whether to do a local lookup or an RPC.
+ * If the querying node is the same as the queried node, it will stay local.
+ * 
+ * @argument id value being searched
```

Comment:
For JavaScript, annotations like this use jsdocs, which has slightly different syntax. Can you either correct to that format or add a follow up issue? https://jsdoc.app/index.html